autoPyTorch/optimizer/smbo.py

```diff
@@ -205,6 +219,23 @@ def __init__(self,
                                               port=self.logger_port)
         self.logger.info("initialised {}".format(self.__class__.__name__))
 
+        self.initial_configurations: typing.Optional[typing.List[Configuration]] = None
+        if portfolio_selection is not None:
+            # read and validate initial configurations
+            portfolio_path = portfolio_selection if portfolio_selection != "greedy" else \
+                os.path.join(os.path.dirname(__file__), '../configs/greedy_portfolio.json')
+            initial_configurations = json.load(open(portfolio_path))
```

Comment:
Can you add an if to check if the provided path does not exists and notify the user with an error?

Reply:
sure